src/components/Pressable/Pressable.tsx

```diff
@@ -256,8 +266,12 @@ export default function Pressable(props: PressableProps) {
       ? props.children({ pressed: pressedState })
       : props.children;
 
+  const flattenedStyles = StyleSheet.flatten((styleProp ?? {}) as ViewStyle);
```

Comment:
Is this cast needed here?

Reply:
removed in [4846fd7](https://github.com/software-mansion/react-native-gesture-handler/pull/2982/commits/4846fd7832f7144eecdfb52ca10058641b0aca68) along with one other unnecessary type cast